### PR TITLE
164 backup

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
Add 3 steps markup
Add navigate between them
#### Where should the reviewer start?
views/Backup/Backup.vue
#### How should this be manually tested?
http://localhost:8080/#/
#### Screenshots (if appropriate)
image
image
image

#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/164-backup

### CODE REVIEW

File: src/views/Backup/Backup.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      class="backup-close"
+      width="12"
+      height="12"
+      color="rgba(255,255,255,0.5)"
+      name="cancel"
+      @click.native="expanded = false"
+    />
+    <BackupStep1
+      v-if="currentStep === 1"
+      @change="onChangeStep"
```
Comment (roma219) [inline src/views/Backup/Backup.vue:22]: This is not a `change` event, isn't it? Better cal it `next` or something like this

File: src/views/Backup/BackupPhrase.vue [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      <div class="backup-step-word">checken</div>
+      <div class="backup-step-word">bitcoin</div>
+      <div class="backup-step-word">money</div>
+      <div class="backup-step-word">car</div>
+      <div class="backup-step-word">city</div>
+      <div class="backup-step-word">children</div>
+      <div class="backup-step-word">carrot</div>
+    </div>
+    <div class="backup-step-content-item">We’ll confirm on the next screen</div>
+    <div class="backup-step-button">i’ve written it down</div>
```
Comment (roma219) [inline src/views/Backup/BackupPhrase.vue]: For button please you the `Button` component - it has all the necessary styles

File: src/views/Backup/Backup.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    BackupPhrase
+  },
+  data() {
+    return {
+      currentStep: 1
+    }
+  },
+  methods: {
+    onChangeStep(step) {
+      this.currentStep = step
```
Comment (roma219) [inline src/views/Backup/Backup.vue:93]: You can just do `this.currentStep++` since you only switch to next step.

**Review by roma219 — CHANGES_REQUESTED** [review]
Ok, needs a few minor changes

File: src/views/Backup/Backup.vue (hunk 1 above)
Comment (lapinlapin) [reply src/views/Backup/Backup.vue:22]: steps can be backward and can be forward. Next does not fit

File: src/views/Backup/BackupPhrase.vue (hunk 2 above)
Comment (lapinlapin) [reply src/views/Backup/BackupPhrase.vue]: I made this in #170

File: src/views/Backup/Backup.vue (hunk 3 above)
Comment (lapinlapin) [reply src/views/Backup/Backup.vue:93]: in my case, one value for both forward and backward. Do not need increment/decrement.

**Review by roma219 — APPROVED** [review]

**Pull request merged**